Bottom: 24d4cbe70a9b7ac06e879df76a8bd04ad0c22499
Top:    6f9687b5adf2d91801467bb70c9e57c7051ca86d
Author: scott snyder <sss@karma>
Date:   2020-04-05 00:09:44 -0400

Refresh of cad-cah-smbc

---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyComic.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyComic.java
index 305d052..43a7672 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyComic.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comictypes/DailyComic.java
@@ -9,6 +9,7 @@ import com.blogspot.applications4android.comicreader.core.Bound;
 import com.blogspot.applications4android.comicreader.core.Comic;
 import com.blogspot.applications4android.comicreader.core.Strip;
 
+
 /**
  * This class is useful for all those comics which are released daily and are indexed
  * by calendar date.
